rpms/picgo/picgo.spec

```diff
@@ -0,0 +1,59 @@
+# Note: Npm depends on network. Network should be enabled when building this software.
+
+%global debug_package %{nil}
+
+%global forgeurl https://github.com/Molunerfinn/PicGo
+
+Version: 2.3.0
+%forgemeta
+
+Name:    picgo
+Release: 1%{?dist}
+Summary: A simple & beautiful tool for pictures uploading built by vue-cli-electron-builder.
+License: MIT
+URL:	 https://molunerfinn.com/PicGo/
+Source0: %{forgesource}
+Source1: picgo.desktop
+Source2: picgo-launcher.sh
+
+Patch0:  picgo-2.3.0-use-electron16.patch
```

Comment:
This is really a big patch. Does it work if you temporary use a target git commit as the source code instead of a released version? That will make maintenance much easier.